services/order.py

```diff
@@ -0,0 +1,28 @@
+from db.models import MovieSession, Order, Ticket
+from django.contrib.auth import get_user_model
+from django.db import transaction
+
+
+@transaction.atomic
+def create_order(
+    tickets: list[dict], username: str, date: str | None = None
+) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+    if date:
+        order.created_at = date
+        order.save()
+    for ticket in tickets:
+        movie_session = MovieSession.objects.get(id=ticket["movie_session"])

```

Comment:
Consider handling the `DoesNotExist` exception for `MovieSession.objects.get(id=ticket['movie_session'])` to avoid runtime errors if the movie session with the given ID does not exist.